test-end-to-end/test-descriptions/001_005_admin_can_remove_content_and_download_again_test_description.md

```diff
@@ -0,0 +1,29 @@
+# Admin can remove content and download it again
```

Comment:
The content isn't being deleted here. Is this scenario testing anything not already tested in the content add scenarios?